guides/v2.2/javascript-dev-guide/javascript/custom_js.md

```diff
@@ -142,10 +142,14 @@ For information about how to initialize your custom JS component in a `.phtml` t
 
 ## Disable default Magento JS {#disable_default_js}
 
-To disable the auto-loading of default Magento JS components and widget
-initialization:
+To disable the auto-loading of default Magento JS components and widget initialization:
+
+1. Create a `requirejs-config.js` file with the following content:
+
+```javascript
```

Comment:
Does this need to be indented to appear correctly?

Reply:
Done